the library doesn't even compile: error[E0063] missing fields in initializer

Genuinely flabbergasted on how this could even happen. Happens whatever code I use with whatever features I use because it happens during compilation of the library. I'm on MacOS 15.4.1, rustc 1.85 if that helps.

```
   Compiling sdl3 v0.14.26
error[E0063]: missing fields `integer_x` and `integer_y` in initializer of `SDL_MouseWheelEvent`
    --> /Users/jmanczak/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/sdl3-0.14.26/src/sdl3/event.rs:1247:29
     |
1247 |                 let event = SDL_MouseWheelEvent {
     |                             ^^^^^^^^^^^^^^^^^^^ missing `integer_x` and `integer_y`

For more information about this error, try `rustc --explain E0063`.
error: could not compile `sdl3` (lib) due to 1 previous error
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -27,14 +27,14 @@ libc = "0.2.169"
 lazy_static = "1.4.0"
 
 [dependencies.sdl3-sys]
-version = "0.4.7+SDL3-3.2.10"
+version = "0.5"
 
 [dependencies.sdl3-image-sys]
-version = "0.1"
+version = "0.2"
 optional = true
 
 [dependencies.sdl3-ttf-sys]
-version = "0.1"
+version = "0.2"
 optional = true
 
 [dependencies.c_vec]

--- a/src/sdl3/event.rs
+++ b/src/sdl3/event.rs
@@ -1254,6 +1254,8 @@ impl Event {
                     direction: direction.into(),
                     mouse_x,
                     mouse_y,
+                    integer_x: 0,
+                    integer_y: 0,
                     reserved: 0,
                 };
                 unsafe {

